app/views.py

```diff
@@ -161,7 +162,7 @@ def handle_app(appname):
                             for item in config['R53']:
                                 print('Getting records for %s' % item['Name'])
                                 ret[item['Name']] = {}
-                                ret[item['Name']]['last_modified']=[str(awslib.get_file_date(bucket_name, file_path))]
+                                ret[item['Name']]['last_modified']=modified_date
```

Comment:
One small change - I'd rather initially set modified_date to None and then only include it if it's not None 🙂